readme.adoc

```diff
@@ -583,7 +583,20 @@ Make sure `kubectl` CLI is installed and configured for the Kubernetes cluster.
 
 === Deployment: Kubepack
 
-https://github.com/aws-samples/aws-microservices-deploy-options/issues/63
+Make sure `kubectl` and `kubepack` CLI is installed and configured for the Kubernetes cluster. To install `kubepack` cli, run:
+
+```
+wget -O kubepack https://github.com/kubepack/kubepack/releases/download/0.1.0/kubepack-darwin-amd64 \
+  && chmod +x kubepack \
+  && sudo mv kubepack /usr/local/bin/
+```
+
+. Move to package root directory: `cd apps/k8s/kubepack`
+. Pull dependent packages: `pack dep -f .`
+. Compile final package: `pack up -f .`
+. Install package: `./manifests/output/install.sh`
```

Comment:
There is no `output` directory.

Directory structure is:

```
manifests
manifests/app
manifests/app/greeting.deployment.yml
manifests/app/greeting.svc.yml
manifests/app/name.deployment.yml
manifests/app/name.svc.yml
manifests/app/webapp.deployment.yml
manifests/app/webapp.svc.yml
manifests/vendor
```

Previous command generated the output:

```
kubepack $ ../../../kubepack dep -f .
3595648 bytes downloaded.
I0404 20:16:56.472399   30361 logs.go:19] Tempdir:  /var/folders/cv/yxmfk09x0ln_pp53287fk775lgvpjt/T/gps-repocache737188486
```

Reply:
@arun-gupta Did you use `pack up -f .` command?

Reply:
The binary is `kubepack` and the output is shown above already. It did not generated the `output` folder.

Reply:
```
# this command gets the dependencies, if you have any
kubepack $ ../../../kubepack dep -f .
```
In your case, `$ ../../../kubepack up -f .` should generate output folder in `manifest` directory. This command generate final combination of your yamls and deploy script in `manifests/output/install.sh`

P.S. We're updating your requested changes above